integrations/common/envs.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -e
+
+readonly IFS=$'\n\t'
```

Comment:
not sure why this file is required, cant we just push these chances into `utils.sh`.
- also utils.sh should be called _helpers.sh to make this consistent with admiral syntax